source/braille.py

```diff
@@ -380,6 +387,52 @@
 BLUETOOTH_PORT = ("bluetooth", _("Bluetooth"))
 
 
+class formattingSymbol(NamedTuple):
+	attribute: str
+	startSymbol: str
+	endSymbol: str
+
+
+fontAttributeBrailleTags: tuple[formattingSymbol] = (
+	formattingSymbol(
+		attribute="bold",
+		# Translators: Brailled at the start of bold text.
+		# This is the English letter "B" in braille.
+		startSymbol=pgettext("braille formatting symbol", "⠃"),
+		# Translators: Brailled at the end of bold text.
+		# This is the English letter "B" plus dot 7 in braille.
+		endSymbol=pgettext("braille formatting symbol", "⡃"),
+	),
```

Comment:
I'm not sure I fully understand the attribute value. Is it just used to identify format symbols?
What do you think about these data-structures instead?
I think if `attribute` is a key to find start/end symbols, it should be reflected in the data-structure.

```suggestion
class FormattingMarker(NamedTuple):
	start: str
	end: str


fontAttributeBrailleTags: dict[str, FormattingMarker] = {
	"bold": FormattingMarker(
		# Translators: Brailled at the start of bold text.
		# This is the English letter "B" in braille.
		start=pgettext("braille formatting symbol", "⠃"),
		# Translators: Brailled at the end of bold text.
		# This is the English letter "B" plus dot 7 in braille.
		end=pgettext("braille formatting symbol", "⡃"),
	),
```

```suggestion
class FormattingMarker(NamedTuple):
	start: str
	end: str

class FormattingSymbol(FormattingMarker, Enum):
	BOLD = FormattingMarker(
		# Translators: Brailled at the start of bold text.
		# This is the English letter "B" in braille.
		start=pgettext("braille formatting symbol", "⠃"),
		# Translators: Brailled at the end of bold text.
		# This is the English letter "B" plus dot 7 in braille.
		end=pgettext("braille formatting symbol", "⡃"),
	),
```